go/vt/proto/.gitattributes

```diff
@@ -0,0 +1,2 @@
+.gitattributes
+*/*.pb.go -diff
```

Comment:
is there a reason we can't put these in the top `.gitattributes` file? (same question for the web/vtadmin one)

Reply:
I had tried to put the directives in the root `.gitattributes` initially and it had not worked, but with these separate ones it did. 

I was trying to reproduce it now, but looks like git is caching the old files: even after deleting all the `.gitattributes` files in the repo `git diff` is still hiding the generated files! 

Tried using `git rm --cached` on the `.gitattributes` files after starting fresh to see if the cache gets deleted, but it still seems to be using the cached ones.  Will try again tomorrow after rebooting the machine ...

Reply:
Thanks @ajm188 @frouioui . Moved to root level. _Seems_ to work, but I have not been able to consistently work around the cached .gitattributes file. Rebooting didn't help, but if put in incorrect file extensions the generated files do show in `git diff`, so presumably having the hints in the root file should work for everyone.